lib/block-supports/typography.php

```diff
@@ -372,12 +374,17 @@ function gutenberg_get_computed_fluid_typography_value( $args = array() ) {
 		)
 	);
 
-	// Protect against unsupported units.
+	// Checks for mandatory min and max sizes, and protects against unsupported units.
```

Comment:
It is, and here's the logic (mine at least 😄)

The min and max font size units can theoretically differ, so one can be `px` and the other `rem`. This is fine in 
the resulting clamp value's first and third args, e.g., `clamp(20px, 1.25rem + ((1vw - 7.68px) * 93.75), 50rem)`

But in order to calculate a linear factor between viewport boundaries we have to do the math on similar values, that is, values whose units are the same. So we coerce the max value unit to whatever the min value unit is.

I use the min value because, if there isn't a min value specified, e.g., it's a custom font size, we use incoming custom font size to set the minimum value, and therefore its unit.

The functions internals are ripe for refactor by the way, but since it's all neatly housed I think we can do it in a follow up.